### PR TITLE
Flying over lava no longer erroneously ignites you ablaze

### LINKED ISSUE #61212: Holoparasite scout mode takes lava damage
Reporting client version: 514.1561

## Round ID:
168918

## Testmerges:
- [Ports pollution from Skyrat (Ported by Gandalf2k15 from Horizon (Made by Azarak))](https://www.github.com/tgstation/tgstation/pull/61154)

## Reproduction:
1) Get a ranged holoparasite.
2) Go to lavaland in mining gear.
3) Have the holoparasite wander over lava a few times.
4) Take lots of damage and probably die.
This seems like unintended behavior, as the scout form is supposed to be able to invisibly and nonharmfully move around like a ghost.

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request

blah blah the refactor added an incorrect if statement
Fixes https://github.com/tgstation/tgstation/issues/61212

## Why It's Good For The Game

why should you be fucking set on fire for flying OVER LAVA

## Changelog

:cl:
fix: Flying over lava no longer erroneously ignites you ablaze.
/:cl:

### CODE REVIEW

File: code/game/turfs/open/lava.dm [hunk 1]
```diff
@@ -151,7 +151,7 @@
 
 /turf/open/lava/proc/can_burn_stuff(atom/movable/burn_target)
 	if(burn_target.movement_type & (FLYING|FLOATING)) //you're flying over it.
-		return isliving(burn_target) ? LAVA_BE_PROCESSING : LAVA_BE_IGNORING
+		return isliving(burn_target) ? LAVA_BE_IGNORING : LAVA_BE_PROCESSING
```
Comment (Fikou) [inline code/game/turfs/open/lava.dm:154]: this makes it so objects that fly can burn is this intended

Comment (SuperNovaa41) [reply code/game/turfs/open/lava.dm:154]: jesus christ, the pre refactor code had it as anything with FLYING or FLOATING wasn't burnt, i should probably just remove the whole statement

**Comment by Colovorat** [comment]
If you get thrown over lava, will you still burn?

**Comment by Watermelon914** [comment]
Yeah you should probably make sure you can't throw people over lava

**Comment by SuperNovaa41** [comment]
image
You can't

**Pull request merged**